### ISSUE TITLE
API key not configurable

### ISSUE THREAD

**Issue by mikayla-maki** [atifazam/vocalizer#14]
After a quick scan of the minified code and the documentation, I don't seen any way to change the NameShouts API key without changing the source. Should this be a `<meta> ` (to continue with the pure markup theme of the API) or should it be part of a new API (e.g. `new Vocalizer(apiKey)`)?

**Comment by atifazam** [comment]
Yeah, it's not ideal that the API key is within the source. I made it this way in order to keep the implementation as simple as possible.

I think putting the API as a `<meta>` would make sense. Want to implement this?

**Comment by mikayla-maki** [comment]
I'd love to work on this!

And you took the words right out of my mouth!

That said, forcing a token to be part of the HTML could make them easy to steal, and it could make a users existing JS token system harder to use (if that system assumes you'll be passing variables to JavaScript.).

**Comment by mikayla-maki** [comment]
Hmmm.... looking into the name shouts API, it seems like the only way to keep the key a secret is to  send the request on the server.

**Comment by atifazam** [comment]
Yeah, maybe we're better off just keeping it the way it is w/ one universal API key? Let me know if you have any other ideas. Thanks for looking into it!
